scripts/s3-utils.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { s3Prefix, s3Bucket } = require('../src/server/s3')
```

Comment:
This file is now in src/server/utils.js it seems.

Reply:
`src/server/config.js`. Fixed...